sign_oca/views/sign_oca_request.xml

```diff
@@ -32,6 +32,8 @@
                 <sheet>
                     <div class="oe_button_box" name="button_box">
                         <button
+                            name="preview"
```

Comment:
Sorry for being late, I think this button should call the action `sign_oca_request_signer_act_window` and add a new button (in the header) with the `preview` action. The change I indicate is added in https://github.com/OCA/sign/pull/8